images/linux/scripts/installers/1604/android.sh

```diff
@@ -37,6 +37,8 @@ else
     exit 1
 fi
 
+# Add required permissions
+chmod -R 755 ${ANDROID_SDK_ROOT}
```

Comment:
Might be better to use `chmod -R a+X` (note capital X) instead of `chmod -R 755`. This way you won't mark files executable that aren't supposed to be executable.

(The capital X modifier says to add the executable permission only to files that already have executable set for some user. It looks like the installed Android SDK binary do have the execute bit set for root.)

Reply:
Thank you for your the suggestion @kentonv, I have tested this solution and it works perfectly fine. I will change the script according to your advice.